scripts/node/utilities.js

```diff
@@ -7,27 +7,42 @@ const { exec } = require('child_process');
 const { promisify } = require('util');
 const winston = require('winston');
 
-const configureWinston = () =>
-  winston.configure({
+const LOG_LEVEL_CONFIG = {
+  error: 'red',
+  warn: 'yellow',
+  success: 'green',
+  info: 'cyan',
+  verbose: 'dim white',
+  debug: 'gray',
+};
```

Comment:
Those are the same with the [default winston levels](https://github.com/winstonjs/winston#logging-levels) (=> npm levels), with the addition of `success` and the removal of `http`. The colors are a bit different to match common terminal script conventions.